pkg/driver_sx126x/contrib/driver_sx126x_hal.c

```diff
@@ -28,10 +28,35 @@
 
 #define ENABLE_DEBUG 0
 #include "debug.h"
+#include "cpu.h"
 
 #define SX126X_SPI_SPEED    (SPI_CLK_1MHZ)
 #define SX126X_SPI_MODE     (SPI_MODE_0)
 
+#ifndef SX126X_PERIPH_WAKEUP_TIME
+#if defined(CPU_FAM_STM32WL)
+#define SX126X_PERIPH_WAKEUP_TIME   10000U
+#else
```

Comment:
Why is it different?

Reply:
The STM32WL [Application note](https://www.st.com/resource/en/application_note/dm00660451-how-to-build-a-lora-application-with-stm32cubewl-stmicroelectronics.pdf) Section 4.3 states RF wakeup time as 10ms.

Reply:
Indeed. I'm not sure though if the old value was right. Maybe we should check that and rather keep these new value.